docs/scos/dev/glue-api-guides/202311.0/conventions/default-conventions.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Default conventions
+description: This document will describe what convention Spryker provides OOTB
+last_updated: December 12, 2023
+template: howto-guide-template
+---
+
+Spryker provides some of the [conventions](/docs/scos/dev/glue-api-guides/{{page.version}}/conventions/what-is-api-convention.html) OOTB
```

Comment:
What does this mean? What the purpose of this sentence? shall the User read the conventions first? before he continuous?

Reply:
Just a cross reference if I opened this page first for some reason. Or you mean that sentence is not clear?